### ISSUE TITLE
[Media Card] Color contrast setting missing CSS

### ISSUE THREAD

**Issue by madalingorbanescu** [pixelgrade/rosa2-lite#36]
In Nova Blocks 1.7.0 we have added color variations for the Media Card, but it looks like those variations are not properly styled in the free version of Rosa2.

![image](https://user-images.githubusercontent.com/46342490/108514814-00aafc80-72cd-11eb-84ff-2c610c21898c.png)

**Comment by madalingorbanescu** [comment]
After some testing, it looks like this issue is related to the way we load some CSS from Nova Blocks. I am going to close the issue here and continue on Nova Blocks repo.
